ui/admin/tests/unit/controllers/scopes/scope/targets/target/add-injected-application-credential-sources-test.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module(
+  'Unit | Controller | scopes/scope/targets/target/add-injected-application-credential-sources',
+  function (hooks) {
+    setupTest(hooks);
+
+    test('it exists', function (assert) {
+      let controller = this.owner.lookup(
```

Comment:
None blocking comment: Can we use `const` here as we did within the test below?

Reply:
Good catch! Thank you!